linebot/models/flex_message.py

```diff
@@ -548,6 +567,8 @@ class SpacerComponent(FlexComponent):
 
     This is an invisible component that places a fixed-size space
     at the beginning or end of the box
+
+    SpacerComponent is deprecated.
```

Comment:
[ask]
Should we keep `SpacerComponent` class? (because its status is discontinued, not deprecated)

Reply:
[fyi]
Some codes related to `SpacerComponent` are still remaining.

https://github.com/line/line-bot-sdk-python/blob/5ab6ba225495248c52eb3aa728da427bedffb37d/tests/models/serialize_test_case.py#L44
https://github.com/line/line-bot-sdk-python/blob/2ff8fe138ef616b0b46696881a6e09afd6871b69/linebot/models/flex_message.py#L323

Reply:
Ok, I think we need to remove `SpacerComponent` class.